src/models/providerEngine/Web3ProviderEngine.js

```diff
@@ -0,0 +1,23 @@
+import {ProviderEngine} from './ProviderEngine';
+import {Web3ProviderEngine as ImportedWeb3ProviderEngine} from '0x.js';
+
+export class Web3ProviderEngine extends ProviderEngine {
+
```

Comment:
remove constructor and do the following
```suggestion
web3ProviderEngine = new ImportedWeb3ProviderEngine()
```